tests/e2e/test_move_task.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+from playwright.sync_api import expect, sync_playwright
+
+from tests.database_test_utils import init_db
+
+
+@pytest.mark.usefixtures("_start_server")
+def test_move_task_to_column() -> None:
+    """
+    Test moving a task in the browser to another column and verify that after a reload
+    it is in the right column.
+    :param start_server: the start server fixture
+    :return: None
+    """
+    init_db(
+        [
+            {"table": "status", "id": 1},
+            {"table": "status", "id": 2},
+            {"table": "status", "id": 3},
+            {"table": "status", "id": 4},
+            {"table": "task", "id": 1, "status_id": 1},
+        ]
+    )
+
+    with sync_playwright() as p:
+        browser = p.chromium.launch()
+        page = browser.new_page()
+        page.goto("http://127.0.0.1:8000/pages/")
+
+        expect(page.locator("#column-1 #card-1")).to_be_visible()
+        expect(page.locator("#column-3")).to_be_visible()
+
+        # todo (Robbert) action below is performed twice, because once does not work, we need find out why and fix it
+        page.locator("#card-1").drag_to(page.locator("#column-3"))
+        page.locator("#card-1").drag_to(page.locator("#column-3"))
+
+        page.reload()
+
+        card = page.locator("#column-3 #card-1")
+        expect(card).to_have_id("card-1")
+        expect(card).to_be_visible()
+
+        browser.close()
```

Comment:
We could create these fixtures:

```
@pytest.fixture(scope="session")
def playwright():
    with sync_playwright() as p:
        yield p

@pytest.fixture(params=["chromium", "firefox", "webkit"])
def browser(playwright, request):
    browser = getattr(playwright, request.param).launch(headless=True)
    context = browser.new_context()
    page = context.new_page()
    yield page
    page.screenshot(path=f"screenshots/{request.param}.png")
    browser.close()
```

Then we can rewrite the test as follows (and test 3 browsers while we're at it)
```
def test_move_task_to_column(start_server, browser) -> None:
    """
    Test moving a task in the browser to another column and verify that after a reload
    it is in the right column.
    :param start_server: the start server fixture
    :return: None
    """
    init_db(
        [
            {"table": "status", "id": 1},
            {"table": "status", "id": 2},
            {"table": "status", "id": 3},
            {"table": "status", "id": 4},
            {"table": "task", "id": 1, "status_id": 1},
        ]
    )

    browser.goto("http://127.0.0.1:8000/pages/")

    expect(browser.locator("#column-1 #card-1")).to_be_visible()
    expect(browser.locator("#column-3")).to_be_visible()

    # todo (Robbert) action below is performed twice, because once does not work, we need find out why and fix it
    browser.locator("#card-1").drag_to(page.locator("#column-3"))
    browser.locator("#card-1").drag_to(page.locator("#column-3"))

    browser.reload()

    card = browser.locator("#column-3 #card-1")
    expect(card).to_have_id("card-1")
    expect(card).to_be_visible()
```

Reply:
If we have `start_server` return the url, we could say: 

```
 browser.goto(f"{start_server}/pages/")
 ```
 
 (PS: maybe we should rename `start_server` to `server`?)

Reply:
This sounds very nice. I used this, but need to find a wait to 'wait for server' now because it gives an async error.. I put I sleep for now, but that is ofcourse a work-around. The tests are running in 3 browsers now though!